### DISCUSSION TITLE
Is histoire actively maintained for Nuxt 4?

### DISCUSSION THREAD

**Discussion by simon-kramer** [histoire-dev/histoire#748] [Q&A]
As described, is it actively maintained and usable for Nuxt 4 (coming up) with its modules, the new structure etc?

**Comment by 50rayn** [comment]
Partly — `1.0.0-beta.1` (Jan 2026) lists improved Nuxt 4 support and is the current `latest`. If it doesn't cover you, [poveste](https://poveste.dev/) is a drop-in fork I maintain with a nuxt4 example book tested on every commit.
